Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1691,7 +1693,8 @@ private void ManageVisibility(bool isRestoringVisibility = false)
             Bounds clippingThresholdBounds = ClippingBoundsCollider.bounds;
 
             Renderer[] contentRenderers = ScrollContainer.GetComponentsInChildren<Renderer>(true);
-            List<Renderer> clippedRenderers = ClipBox.GetRenderersCopy().ToList();
+            clippedRenderers.Clear();
+            clippedRenderers.UnionWith(ClipBox.GetRenderersCopy());
```

Comment:
Not a blocker for this PR but going forward it might be interesting to see if we can get rid of this `ClipBox.GetRenderersCopy()` call too, which looks like it allocates a new `List` in the implementation. If we expect the underlying list to potentially change while we're iterating through this, could we instead set up a lock?